src/FsAutoComplete/FsAutoComplete.Stdio.fs

```diff
@@ -102,24 +104,17 @@ let main (commands: Commands) (commandQueue: BlockingCollection<Command>) =
       |> Async.Catch
       |> Async.RunSynchronously
       |> function
-         | Choice1Of2 res -> res |> List.iter (CommandResponse.serialize writeJson >> Console.WriteLine)
+         | Choice1Of2 res -> res |> List.iter (CommandResponse.serialize serializer >> Console.WriteLine)
          | Choice2Of2 exn ->
             exn
             |> sprintf "Unexpected internal error. Please report at https://github.com/fsharp/FsAutoComplete/issues, attaching the exception information:\n%O"
-            |> Response.error writeJson
+            |> Response.error serializer
             |> Console.WriteLine
     0
 
-let trimBOM (s: string) =
-    let bomUTF8 = Text.Encoding.UTF8.GetString(Text.Encoding.UTF8.GetPreamble())
-    if (s.StartsWith(bomUTF8, StringComparison.Ordinal)) then
-        s.Remove(0, bomUTF8.Length)
-    else
-        s
-
-let start (commands: Commands) (args: Argu.ParseResults<Options.CLIArguments>) =
+let start config (commands: Commands) (args: Argu.ParseResults<Options.CLIArguments>) =
     Console.InputEncoding <- Text.Encoding.UTF8
-    Console.OutputEncoding <- new Text.UTF8Encoding(false, false)
+    Console.OutputEncoding <- Text.Encoding.UTF8
```

Comment:
I did some reading and .net core has long-since changed to a default utf8 without BOM encoding, so I purged this workaround while I was here.